### PR TITLE
[IMP]10710 add module pos_membership

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: pos_membership/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline pos_membership/__openerp__.py]: Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: pos_membership/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{'name': 'Membership Management - POS Membership',
+ 'version': '1.0',
```
Comment (elicoidal) [inline pos_membership/__openerp__.py]: version 8.0.1.0.0

File: pos_membership/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+ 'depends': [
+     'membership_account_balance',
+     'point_of_sale',
+     'account_accountant',
+     'account_voucher'
+ ],
+ 'author': 'Elico Corp',
+ 'license': 'AGPL-3',
+ 'website': 'https://www.elico-corp.com',
+ 'description': """
```
Comment (elicoidal) [inline pos_membership/__openerp__.py]: remove => README

File: pos_membership/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization
+    whose mission is to support the collaborative development of Odoo features
+        and promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
+""",
+ 'images': [],
```
Comment (elicoidal) [inline pos_membership/__openerp__.py]: Remove empty keys

File: pos_membership/models.py [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api, _
```
Comment (elicoidal) [inline pos_membership/models.py]: Alphabetical order for the imports

**Comment by elicoidal** [comment]
It should be PEP8 but this module should be proposed directly to OCA

File: pos_membership/__openerp__.py (hunk 3 above)
Comment (Rona111) [inline pos_membership/__openerp__.py]: @elicoidal  you mean delete the description in openerp.py?

Comment (elicoidal) [inline pos_membership/__openerp__.py]: yes: it should appear only in README. Please read carefully the 
guidelines here below:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md

Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

README:
Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Comment by elicoidal** [comment]
@noahzaozao is not it redundant with the work you have done lately?

**Pull request closed without merging**